examples/webpack.config.local.js

```diff
@@ -73,6 +73,19 @@ function makeLocalDevConfig(EXAMPLE_DIR = LIB_DIR, linkToLuma) {
           test: /\.js$/,
           use: ['source-map-loader'],
           enforce: 'pre'
+        },
+        {
+          // Compile source using buble
+          test: /\.js$/,
+          loader: 'buble-loader',
```

Comment:
Well, the inclusion of this rule breaks local builds against luma.gl, since buble cannot parse anything beyond ES6, and luma.gl now uses ES6+ syntax.

Since I didn't encounter this problem when running layer-browser, maybe make the buble injection dependent on whether local luma is enabled for now?

FWIW, this rule was only intended to support JSX compilation.

Reply:
This rule only includes deck.gl directories.